common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js

```diff
@@ -322,58 +239,33 @@ const IIIFPresentationDisplay = ({
               });
             }}
           >
-            <BookPreview
-              columnNumber={
-                itemsNumber === 1
-                  ? 3
-                  : itemsNumber === 3
-                  ? 4
-                  : 2 + Math.floor(itemsNumber / 2)
-              }
-              hasThumbs={imageThumbnails.length > 0}
-            >
-              {imageThumbnails.map((pageType, i) => {
-                return pageType.images.map(image => {
-                  return i === 0 ? (
-                    <PagePreview
-                      key={image.id}
-                      backgroundImage={iiifImageTemplate(image.id)({
-                        size: 'max',
-                      })}
-                    />
-                  ) : (
-                    <PagePreview
-                      key={image.id}
-                      backgroundImage={iiifImageTemplate(image.id)({
-                        size: '!400,400',
-                      })}
-                    />
-                  );
-                });
-              })}
-              <CallToAction
-                hasThumbs={imageThumbnails.length > 0}
-                className={classNames({
-                  [font({ s: 'HNM4' })]: true,
-                })}
-              >
-                <span className="cta__inner">
-                  <span
-                    className={classNames({
-                      'flex-inline': true,
-                      'flex--v-center': true,
+            {imageThumbnails.map((pageType, i) => {
+              return pageType.images.map(image => {
+                return (
+                  <IIIFResponsiveImage
+                    key={image.id}
+                    lang={'en'}
```

Comment:
is `lang` nullable? If so this would probably be better as `null`?

Reply:
yep, good spot, will change